vars/env/prod/env.yml

```diff
@@ -84,7 +84,7 @@ env_host_url: "{{ env_external_dns_name }}"
 
 # django environment specific apps list
 # A test client - moved to aws-test / dev /impl settings
-env_django_environment_apps: ['storages', ]
+env_django_environment_apps: ['storages', 'apps.testclient', ]
 env_django_optional_installed_apps: [ ]
```

Comment:
This **might** also need added here as well.  Not 100% sure but its here for TEST and IMPL too.

Reply:
oh... cool. I will match up then